### ISSUE TITLE
[OUDS] [Tokens] Integration of System OUDS v2.2.0

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/Orange-Boosted-Bootstrap#3208]
# Descriptions

Vertbatims of @MaximeTonnerre 

### References added:
- [ ] color-action-read-only-primary-light→color-repository-opacity-black-higher
- [ ] color-action-read-only-primary-dark→color-repository-opacity-white-high
- [ ] color-action-read-only-secondary-light→color-repository-opacity-black-medium-low
- [ ] color-action-read-only-secondary-dark→color-repository-opacity-white-medium-low
 
### References added:
- [ ] color-border-minimal-light→color-repository-opacity-black-lowest
- [ ] color-border-minimal-dark→color-repository-opacity-white-lowest

## Navigation-bar

Numerous references were added to meet the needs of the component's creation → [Navigation bar tokens repository](https://www.figma.com/design/AGpy41dOJU1kWV2GF22WJX/Navigation---Bar?node-id=16002-35707&t=OaVAEqbyIe9gzFC8-4)
 
### References added:
- [ ] control-control-item-space-padding-block-top-error-text→space-padding-block-xsmall

### References renamed:
- [ ] control-control-item-space-padding-block→control-control-item-space-padding-block-default

[OUDS-system-ouds-V2.2.0.zip](https://github.com/user-attachments/files/23418397/OUDS-system-ouds-V2.2.0.zip)

**Comment by B3nz01d** [comment]
Issue closed during Sprint # 24
